spec/views/hyrax/admin/collection_types/_form_settings.html.erb_spec.rb

```diff
@@ -8,24 +21,47 @@
     end
   end
 
-  before do
-    assign(:form, collection_type_form)
-    allow(view).to receive(:f).and_return(form)
-    render
-  end
+  context "collection_type.collections? is false" do
```

Comment:
Would you consider renaming the `context` to `when collection_type.collections? is false`? The rspec output reads more idiomatically when `context` blocks begin with e.g., `when` or `with`.